src/teb_config.cpp

```diff
@@ -350,6 +352,13 @@ void TebConfig::checkParameters() const
   // weights
   if (optim.weight_optimaltime <= 0)
       ROS_WARN("TebLocalPlannerROS() Param Warning: parameter weight_optimaltime shoud be > 0 (even if weight_shortest_path is in use)");
+
+  // holonomic check
+  if (robot.max_vel_y >= 0) {
+    if (robot.max_vel_linear < robot.max_vel_x || robot.max_vel_linear < robot.max_vel_y) {
+      ROS_WARN("TebLocalPlannerROS() Param Warning: max_vel_linear < max_vel_x or max_vel_linear < max_vel_y. Assuming max(max_vel_x, max_vel_y)");
```

Comment:
S: instead of assuming `max(max_vel_x, max_vel_y)` I think it would be better to stick with the given value and simply print the warning. Vx and Vy would be clamped in `saturateVelocity` to match the given max linear vel, instead of the other way around.